web/public/js/directives/dojo/join-dojo/index.js

```diff
@@ -17,13 +17,13 @@ angular
             $scope.loggedOut = _.isEmpty($scope.currentUser);
             var approvalRequired = ['mentor', 'champion'];
             $scope.ninjaRole = {
-              name: 'Ninja',
+              name: 'Attendee',
```

Comment:
As this is one of the first cases that it is used I wanted to use more general language. Added one sentence to explain what a 'Ninja' is to the first description :)